src/main/java/no/difi/sdp/client2/domain/Databehandler.java

```diff
@@ -1,23 +1,21 @@
 package no.difi.sdp.client2.domain;
 
-import no.digipost.api.representations.Organisasjonsnummer;
-
 public class Databehandler {
 
-    public final Organisasjonsnummer organisasjonsnummer;
+    public final DatabehandlerOrganisasjonsnummer organisasjonsnummer;
     public final Noekkelpar noekkelpar;
 
-    private Databehandler(Organisasjonsnummer organisasjonsnummer, Noekkelpar noekkelpar) {
+    private Databehandler(DatabehandlerOrganisasjonsnummer organisasjonsnummer, Noekkelpar noekkelpar) {
 
         this.organisasjonsnummer = organisasjonsnummer;
         this.noekkelpar = noekkelpar;
     }
 
     /**
      * @param organisasjonsnummer Organisasjonsnummeret til avsender av brevet.
-     * @param noekkelpar Avsenders nøkkelpar: signert virksomhetssertifikat og tilhørende privatnøkkel.
+     * @param noekkelpar          Avsenders nøkkelpar: signert virksomhetssertifikat og tilhørende privatnøkkel.
```

Comment:
Denne lar jeg være, for det er autoformat av kommentaren. Ser jalla ut her, men den aligner beskrivelsen med den over på en fin måte.